### ISSUE TITLE
torch.sparse.softmax missing support for sparse CSR

### ISSUE THREAD

**Issue by samuele-bortolato** [pytorch/pytorch#134396]
### 🚀 The feature, motivation and pitch

sparse.softmax apparently only supports sparse COO format, while sparse CSR is presented as more efficient and default format for other operations. 
For example in my use case I'm trying to implement an efficient sparse attention using torch.sparse.sampled_addmm and sparse.softmax, the problem is that the first only supports sparse CSR.

**Comment by nikitaved** [comment]
We have a Triton SDPA kernel that works with BSR. Could that be a temporary solution to the problem?

**Comment by samuele-bortolato** [comment]
Thank you for the response and sorry for the late answer, I missed the notification email.

I may be able to use that as a temporary workaround, could you please point me the the documentation of such kernel?
I could't quite manage to find it.

Thanks
